app/javascript/edit_form_reply.js

```diff
@@ -0,0 +1,33 @@
+$(document).on('turbo:load', function() {
+  $('.form-edit').each(function() {
+    var formEditId = $(this).attr('id');
```

Comment:
Please check when to use let, var, const. Basically, only let and const should be used.